packages/utils/src/address.ts

```diff
@@ -1,7 +1,35 @@
-import { privateToAddress, toChecksumAddress } from 'ethereumjs-util'
+import assert = require('assert')
+import {
+  ecrecover,
+  fromRpcSig,
+  privateToAddress,
+  privateToPublic,
+  pubToAddress,
+  sha3,
+  toChecksumAddress,
+} from 'ethereumjs-util'
+import Web3 from 'web3'
 
 export type Address = string
 
+export async function addressToPublicKey(address: string, web3: Web3) {
```

Comment:
The `utils` package currently does not specify `web3` as a dependency, thus generally what I did in `signatureUtils` is to pass the `web3.eth.sign` method as a parameter instead. That also highlights to the consumer of this function that there should be a web3 provider that can actually sign (vs. a web3 parameter that is used to make view calls for example).

Reply:
Also would this make sense in signatureUtils instead?